Issue.py

```diff
@@ -15,7 +15,15 @@ def __init__(self, title, labels, assignees, milestone, body, hunk, file_name,
         self.markdown_language = markdown_language
         self.status = status
         self.identifier = identifier
+        self.identifier_actual = identifier_actual
         self.ref = ref
         self.issue_url = issue_url
         self.issue_number = issue_number
+        self.start_line_within_hunk = start_line_within_hunk
 
+    def __str__(self):
+        selflist = []
+        for key in [x for x in vars(self).keys() if x not in ("hunk")]:
+            selflist.append(f'"{key}": "{getattr(self, key)}"')
+        selflist.append((f'"hunk": "{self.hunk}"'))
+        return '\n'.join(selflist)
```

Comment:
This is mainly just for debug, but it's also utilized in the error path of the unit tests. It just provides us a way to `print` an `Issue`